gammapy/data/observations.py

```diff
@@ -342,6 +382,61 @@ def select_time(self, time_interval):
         obs.obs_filter = new_obs_filter
         return obs
 
+    @classmethod
+    def from_file(cls, event_file, irf_file):
+        """Create an Observation from a Event List and IRF file.
+        Ease the creation of a single Observation without creating an entire `DataStore`
+        that is without the need of HDU and OBS index files
+        """
+        log.info(f"reading the Event List and GTI from {event_file}")
+        event_file = make_path(event_file)
+        if not event_file.exists():
+            raise OSError(f"File not found: {event_file}")
+        log.debug(f"Reading {event_file}")
+        events = EventList.read(event_file)
```

Comment:
Here you can directly use the `HDULocation` system. Here is a minimal example:
```python3
from gammapy.utils.fits import HDULocation

loc = HDULocation(
    hdu_class="events",
    base_dir="$GAMMAPY_DATA/hess-dl3-dr1/data",
    file_dir="",
    file_name="hess_dl3_dr1_obs_id_020136.fits.gz"
)

events = loc.load()
```

Reply:
That looks much more complicated than `EventList.read()`, what is the advantage?

Reply:
Agreed. In the case of event lists there is no clear advantage to go via the registry / lazy loading interface here, because there is only one class to represent the data. The main remaining difference is that `HDULocation` triggers the lazy loading. So events are only read when `Observation.events` is accessed. But again in this case the lazy loading does not matter, so indeed one could keep `EventList.read()` as well.